route/route.go

```diff
@@ -32,6 +33,11 @@ type Router struct {
 	Metrics      metrics.Metrics       `inject:""`
 }
 
+type BatchResponse struct {
+	Status int    `json:"status"`
+	Error  string `json:"error"`
```

Comment:
could you add `omitempty` - when there are no errors the API omits the `error` key from the returned JSON object.